app/models/family.rb

```diff
@@ -229,6 +229,9 @@ def self.mappable_details
       "where people.visible = #{Family.connection.quoted_true} " \
       "and families.visible = #{Family.connection.quoted_true} " \
       "and families.deleted = #{Family.connection.quoted_false} " \
+      "and (people.share_address = TRUE " \
+      "  or  #{Person.logged_in.admin?(:view_hidden_profiles)} " \
+      "  or  people.id = #{Person.logged_in.id} )" \
```

Comment:
Let's do this instead:

```
(Person.logged_in.admin?(:view_hidden_attributes) ? '' : "and people.share_address = #{Family.connection.quoted_false}")
```

1. I changed `:view_hidden_profiles` to `:view_hidden_attributes`, which is the admin privilege that decides if an admin can see non-shared stuff.
2. I took out the exception if the user is viewing their own family -- they shouldn't see their point on the map because they will think other people can see it too.